docs/open-source-tizen/tizen-rt/rt-specification-1-0.md

```diff
@@ -1,120 +1,128 @@
 # Tizen RT 1.0 Specification
```

Comment:
Bullet lists under "File system support", "Database (AraStorage) Support", "Feature support in IoTivity 1.2" and "Device Management Services" were missing. We restored them from TD. Is that OK?

Reply:
Good job.